source/agora/network/Manager.d

```diff
@@ -1366,7 +1366,8 @@ public class NetworkManager
     public void whitelist (Hash utxo)
     {
         this.peers[].filter!(p => p.utxo == utxo)
-            .each!(p => p.client.address.each!(addr => this.banman.whitelist(addr)));
+            .each!(p => p.client.address.filter!(a => a != Address.init)
```

Comment:
```suggestion
            .each!(p => p.client.address.filter!(a => a !is Address.init)
```

Otherwise it might call `opEquals` anyways.

Reply:
Updated